discord_connector.py

```diff
@@ -277,6 +279,78 @@ def _handle_ban_user(self, param):
 
         return status
 
+    def _handle_fetch_message_history(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        channel_id = param['channel_id']
+
+        status_after, fetching_start_date = self.parse_date(param.get('fetching_start_date', None))
+        status_before, fetching_end_date = self.parse_date(param.get('fetching_end_date', None))
+
+        limit = param.get('limit', None)
+        if limit == 0:
+            limit = None
+        oldest_first = param.get('oldest_first', False)
+
+        self.save_progress(
+            "status_after {}: {} | status_before {}: {}".format(status_after, fetching_start_date, status_before,
+                                                                fetching_end_date))
+        if not (status_after and status_before):
+            return action_result.set_status(phantom.APP_ERROR,
+                                            "action result: fetching messages from {} channel ended with failure, unable to format date"
```

Comment:
why not formatting with `f-string`? 
same in :308

Reply:
It seams that .format() is standardized way of handling strings, however it may be because it is older than f-string itself. I can change those to f-string if there is a need for that.